src/main/res/values/strings.xml

```diff
@@ -122,5 +122,6 @@
     <string name="location_permission_request_explanation">WhereYouGo needs access to the GPS on your device to locate your position and calculate distance and direction to the Cartridge locations.</string>
     <string name="storage_permission_request_explanation">WhereYouGo will write data onto your phone storage or SD card as soon as you save the Cartridge.</string>
     <string name="permission_conclusion">The app cannot be used without this permission(s).</string>
+    <string name="dialog_no_password">You havn\'t configured a password and/or username! Please do so in the WhereYouGo Settings. Click OK to open the WhereYouGo settings now. Click Cancel to close this dialog and return to the previous screen.</string>
```

Comment:
The user needs password and username.
Please remove "or".

Reply:
Well the user can enter only a username or a password. There is no check currently that the user has to enter both values in the Activity. Since this may be the case I decided to have the "or" there. If you still think only "and" needs to be there I will of course remove it. I am aware that is unlikely that only one value is configured.

Reply:
You are right - I had to read the code once more.

A litte improvement idea. I suggest to change the first sentence to
"You have not configured your user name and/or password! "

Reply:
Should be fixed by 430e0a7